app/views/partials/personal_key/_key.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="bg-light-blue bg-personal-key">
-  <p class="bold center margin-top-1 padding-top-1 fs-20p">
+  <h2 class="bold center margin-top-1 padding-top-1 fs-20p">
```

Comment:
This will change the font from a sans-serif to a serif, which I don't know that we'd want to change ([screenshot](https://user-images.githubusercontent.com/1779930/132745997-edcd0097-61fd-45b4-8565-b598329a7f1e.png), cc @anniehirshman-gsa).

We could maintain the visual appearance using a [font-family utility](https://designsystem.digital.gov/utilities/font-size-and-family/#utility-font-family) ([screenshot](https://user-images.githubusercontent.com/1779930/132746396-bcb40cbc-9e98-416c-b44e-4f91b18e0f7e.png)). Also, since headings are already bold by default, we could remove the extra `bold` class.

```suggestion
  <h2 class="center margin-top-1 padding-top-1 fs-20p font-family-body">
```

Reply:
@aduth @mdiarra3 I'm OK with this `h2` changing to a serif for the moment to avoid extra styling for this heading... since this will change back to a sans anyway with the Public Sans update (see screenshot).  Appreciate the heads up though!
![Screen Shot 2021-09-09 at 5 31 21 PM](https://user-images.githubusercontent.com/66273797/132765886-dba68f19-b09e-4a71-9bd6-3d2235170b19.png)